File: classObj.cpp
#include <iostream>
#include <string>

using namespace std;

class Brand {
public:
    
    string name;
    string color;
    int price;

    // Constructor Declearation
    Brand(string x, string y, int z) {
        name = x;
        color = y;
        price = z;
    }

    
    void displayDetails() {
        cout << "Brand Name: " << name << ", color: " << color << ", price: " << price << endl;
    }
};

int main() {
    
    Brand myShirt("T-shirt", "White",200);

   
    myShirt.displayDetails();

    return 0;
}

File: classObj.py
class Brand:
    
    def __init__(self, name, color, price):
        self.name = name
        self.color = color
        self.price = price

   
    def display_details(self):
        print(f"Brand name: {self.name}, color: {self.color}, price: {self.price}")


myShirt = Brand("T-shirt", "Red", 200)


myShirt.display_details()
